questions/qa-date-format.en.html

```diff
@@ -56,21 +59,83 @@ <h1>Date formats</h1>
-    it. Resist this impulse. Do you really want to keep separate copies of documents for the U.S. and the U.K. that differ only in date format? In any
-    case you still have to deal with multilingual users like the one in our example above.</p>
-  <p>You have three options to consider, all with advantages and drawbacks:</p>
+  <p>
+    There are mainly two aspects related to date processing on the Web: <a href="#input">how the user enters a date</a> and <a href="#display">how the author displays a date</a>.
+  </p>
+
+	<section>
+    <h2 id="input">User input</h2>
+
+    <p>
+      When it comes to accept data from the user, we usually use the <code class="kw" translate="no">input</code> element in HTML.
+      HTML5 introduced <code class="kw" translate="no">input</code> elements of type <code>date</code>, <code>time</code>, <code>datetime-local</code>, <code>month</code>, and <code>week</code>, which let the user easily enter a date or a time, like:
+    </p>
+
+    <div style="margin-inline-start: 25px;">
+      <p>
+    <label for="date">&lt;input type="<strong>date</strong>"&gt;</label>
+      <input type="date" id="date" name="date"
+       value="2021-12-07"
+       min="2021-12-05" max="2021-12-10">
+    </p>
+      <p>
+       <label for="time">&lt;input type="<strong>time</strong>"&gt;</label>
+      <input type="time" id="time" name="time"
+       min="09:00" max="18:00" required>
+      </p>
+      <p>
+       <label for="datetime-local">&lt;input type="<strong>datetime-local</strong>"&gt;</label>
+      <input type="datetime-local" id="datetime-local"
+       name="datetime-local" value="2021-12-07T18:00"
+       min="2021-12-05T00:00" max="2021-12-10T00:00">
+      </p>
+      <p>
+       <label for="month">&lt;input type="<strong>month</strong>"&gt;</label>
+      <input type="month" id="month"
+       name="month" value="2021-01"
+       min="2021-01">
+      </p>
+      <p>
+       <label for="week">&lt;input type="<strong>week</strong>"&gt;</label>
+      <input type="week" id="week"
+       name="week" min="2021-W01"
+       min="2021-W05">
+      </p>
+    </div>
+    <p>
+      Each browser has its own way of rendering these elements, and browsers that do not support them will fall back to a single-line text field.
+    </p>
+
+    <p>
+      <a href="https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#dates-and-times">HTML</a> uses a variation of the <a href="https://en.wikipedia.org/wiki/ISO_8601">ISO 8601</a> standard for its date and time strings. The date and time strings do not include the time zone data. The time zone data can be stored on the server-side separately if needed.
+    </p>
+
+    <p>
+      You can set a default value for the input by using the <code>value</code> attribute or restrict the date or time using the <code>min</code> and <code>max</code> attributes.
```

Comment:
And step.  What about 12 vs 24 hour clock settings?

Reply:
Should we mention the autocomplete features?

Reply:
This is part of the 'storage' topic: if you set a default value, it needs to follow the ISO format, not the local order/composition. We should describe that, i think.

You may find some inspiration from https://html.spec.whatwg.org/multipage/forms.html#input-author-notes (especially the latter half of that subsection).